File: Day17/evenandodd.java
import java.util.Scanner;
public class Count
{
	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		int even=0;
		int n= s.nextInt();
		int arr[] = new int[n];
		for(int i=0;i<n;i++){
		    arr[i] = s.nextInt();
		    if(arr[i]%2==0){
		        even++;
		    }
		} 
		System.out.println(even+" "+(n-even));
	}
}

File: Day17/evenandodd.c
#include <stdio.h>

int main()
{
    int n,odd=0,even=0;
    int arr[100];
    scanf("%d ",&n);
    
    for(int i=0;i<n;i++){
        scanf("%d",&arr[i]);
        if(arr[i]%2!=0){
            odd++;
        }
        else{
            even++;
        }
        
    }
    printf("%d %d",even,odd);
    

    return 0;
}
